packages/webpack-loader/src/extract-plugin.ts

```diff
@@ -108,6 +108,13 @@ export class CompiledExtractPlugin {
 
   constructor(options: CompiledExtractPluginOptions = {}) {
     this.#options = options;
+
+    // @ts-expect-error -- Make sure this config doesn't bleed in as it's passed through
+    if (options.classHashPrefix) {
+      throw new Error(
+        '`@compiled/webpack-loader.CompiledExtractPlugin` is mixing the `extract: true` and `classHashPrefix` options, which is not supported and will result in bundle size bloat.'
+      );
+    }
```

Comment:
This may not work given the `classHashPrefix` is passed to the babel plugin isn't it?  I'm a bit uncertain how this works to be frank!